pkg/secrets/plugins/host/plugin.go

```diff
@@ -22,6 +24,10 @@ func (p Plugin) Close() error {
 	return nil
 }
 
+func (p Plugin) Create(keyName string, keyValue string, value string) error {
+	return fmt.Errorf("the %s plugin does not support saving secrets. Please configure a secret plugin", PluginKey)
```

Comment:
This error message doesn't give the user enough information to know how to fix the problem. They would need to then dig around on the site and hope that somewhere we explained what to do.

Whenever we know how to resolve an error, we should always link to the relevant documentation page. Or if there isn't a doc page (yet), tell them how to fix it in the message like "Edit your porter config file and set default-storage-plugin to X to use the insecure filesystem plugin. Do not use this plugin for production data."

Reply:
Thanks for the recommended error message! I have updated it in the code